tasks/gulp/copy-to-destination.js

```diff
@@ -26,11 +26,11 @@ gulp.task('copy-files', () => {
     '!' + configPaths.src + 'globals/icons',
     '!' + configPaths.src + 'globals/scss',
     '!' + configPaths.src + 'globals',
-    '!' + configPaths.src + 'views',
     '!' + configPaths.src + 'components',
-    '!' + configPaths.src + 'views/**',
-    '!' + configPaths.src + 'examples',
-    '!' + configPaths.src + 'examples/**',
+    '!' + configPaths.app,
```

Comment:
We start out with `configPaths.src`, so all of app should be excluded already anyway?

Reply:
Ha, good one. Thanks!